docs/connecting-to-a-lit-network/lit-blockchains/chronicle.md

```diff
@@ -36,10 +36,10 @@ To connect to Chronicle, you can click <AddRollupButton /> or manually add the n
 | Name               | Chronicle - Lit Protocol Testnet               |
 | RPC URL            | https://chain-rpc.litprotocol.com/replica-http |
 | Block Explorer URL | https://chain.litprotocol.com/                 |
-| Currency Symbol    | testLPX                                        |
+| Currency Symbol    | tstLPX                                        |
 | Currency Decimals  | 18                                             |
 
```

Comment:
see comment above. Since this is for the old Chronicle, i believe the ticker should be listed as 'tstLIT'

Reply:
Sorry about this, changed it to `tstLIT`